documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -240,10 +240,14 @@ Pessimistic transactions are more costly by their nature: each write operation p
 The write skew anomaly occurs when 2 transactions read and update the same key and both of them can commit successfully without having seen the update performed by the other.
 To detect and rollback one of the transaction, `write-skew` should be enabled.
 
-NOTE: The write skew check is only performed for `REPEATABLE_READ` isolation.
+[NOTE]
+====
+* The write skew check is only performed for `REPEATABLE_READ` isolation.
 
-NOTE: Pessimistic transaction does not perform any write skew check.
-It can be avoided by locking the key at read time. Look how at the example below.
+* Pessimistic transactions do not need to perform write skew checks if they lock keys at read time with `Flag.FORCE_WRITE_LOCK`.
```

Comment:
This wording is confusing: in pessimistic mode there's no WSC functionality, neither when you don't lock the keys explicitly - you're just left with stale reads.